js/main.js

```diff
@@ -1,2 +1,214 @@
+    //console.log to see if it works
+    console.log('user wins')
+    //declare result
+    result = 'win! :)'
+  } else if(userPick === 'scissors' && botPick === 'rock'){
+    //console.log to see if it works
+    console.log('bot wins')
+    //declare result
+    result = 'lose :('
+  } else if(userPick === 'paper' && botPick === 'scissors'){
+    //console.log to see if it works
+    console.log('bot wins')
+    //declare result
+    result = 'lose :('
+  } else if(userPick === 'paper' && botPick === 'rock'){
+    //console.log to see if it works
+    console.log('user wins')
+    //declare result
+    result = 'win! :)'
+  } else{
+    //console.log to see if it works
+    console.log('something went wrong')
+  }
+}
+//scoreKeeper function
+function updateScore(){
+  if (result === 'win! :)'){
+    // console.log('user wins')
+    userScore += 1
+    $('#humanScore').text(`${userScore}`)
+  } else if (result === 'lose :('){
+    // console.log('bot wins')
+    botScore+= 1
+    $('#computerScore').text(`${botScore}`)
+  } else if (result === 'tie!'){
+    // console.log('tie!')
+  } else {
+    // console.log('something went wrong')
+  }
+}
+
+// messages if won, lost
+function updateMessage(){
+  //You played ... the bot played... you lose!, win :), tie
+  $('#status').html(`You played <strong>${userPick}</strong>. The bot played <strong>${botPick}</strong>. You <strong>${result}</strong>`)
+  //figure out how to bold this text
+  // $('userPick', 'botPick', 'result').addClass('boldText')
+}
+
+//generate botPick using random number generator and assigning them to a botPick
+function botPicker(){
+  let randomNumber = Math.floor(Math.random() * 3) + 1
+  console.log(randomNumber)
+  if(randomNumber === 1){
+    botPick = 'rock'
+  } else if(randomNumber === 2){
+    botPick = 'paper'
+  } else if (randomNumber === 3){
+    botPick = 'scissors'
+  } else {
+    console.log('something went wrong')
+  }
+}
+//BONUS 1
+
+//end of round message function
+function endOfRoundMessage(){
+  $('#status').text(`The winner of this round is ${winnerOfRound} :)!`)
+}
+function updateRoundScore(){
+  if (userScore === 5){
+    // console.log('user wins round')
+    userRoundScore += 1
+    $('#humanRound').text(`${userRoundScore}`)
+    //reset Scoreboard
+    userScore = 0
+    botScore = 0
+    $('#humanScore').text(`${userScore}`)
+    $('#computerScore').text(`${botScore}`)
+    winnerOfRound = 'user'
+    endOfRoundMessage()
+  } else if (botScore === 5){
+    // console.log('bot wins')
+    botRoundScore+= 1
+    $('#computerRound').text(`${botRoundScore}`)
+    userScore = 0
+    botScore = 0
+    $('#humanScore').text(`${userScore}`)
+    $('#computerScore').text(`${botScore}`)
+    winnerOfRound = 'bot'
+    endOfRoundMessage()
+  } else{
+    // console.log('tie!')
+    // console.log('something went wrong')
+  }
+}
+
+
+//BONUS 2
+function gameOver(){
+  if(userRoundScore === 2 || botRoundScore === 2){
+    console.log('end of game')
```

Comment:
Your gameOver function definitely works! But instead of console.log "end of game", you could have it display the winner in the scoreboard for instance. [Here's](https://github.com/jsd20200127/rock-paper-scissors-solution/blob/502d85718e17749df951e7808338ada1ec6d1844/js/main-with-bonus.js#L128) an alternative sample solution for that alerts the user.